lib/_layouts/_article.njk

```diff
@@ -25,8 +25,11 @@
     }
   }) %}
     <div class="nhsbsa-sticky-nav">
-      <h2 class="nhsbsa-sticky-nav__heading">Contents</h2>
-      {{ content | toc | safe }}
+      {% if content | toc %}
+        <h2 class="nhsbsa-sticky-nav__heading">Contents</h2>
+        {{ content | toc | safe }}
+      {% endif %}
+      &nbsp;
```

Comment:
@GeorgeRowley this nbsp is my hack to keep the content navigation space in place when we have no table of contents. Can you fix it please?

Reply:
Hi Pat, sure thing 👍🏼